doc/esmvalcore/config.rst

```diff
@@ -4,29 +4,163 @@
+  compress_netcdf: false
+
+  # Save intermediary cubes in the preprocessor true/[false]
+  # set to true will save the output cube from each preprocessing step
+  # these files are numbered according to the preprocessing order
+  save_intermediary_cubes: false
+
+  # Remove the preproc dir if all fine
+  # this option true will remove ALL preprocessor files
+  # CAUTION when using: if you need those files, set it to false
+  remove_preproc_dir: true
+
+  # Run at most this many tasks in parallel null/[1]/2/3/4/..
+  # Set to null to use the number of available CPUs.
+  # Make sure your system has enough memory for the specified number of tasks.
+  max_parallel_tasks: 1
+
+  # Path to custom config-developer file, to customise project configurations.
+  # See config-developer.yml for an example. Set to None to use the default
+  config_developer_file: null
+
+  # Get profiling information for diagnostics
+  # Only available for Python diagnostics
+  profile_diagnostic: false
+
+  # Rootpaths to the data from different projects (lists are also possible)
+  rootpath:
+    CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2]
+    OBS: ~/obs_inputpath
+    default: ~/default_inputpath
+
+  # Directory structure for input data: [default]/BADC/DKRZ/ETHZ/etc
+  # See config-developer.yml for definitions.
+  drs:
+    CMIP5: default
 
+Most of these settings are fairly self-explanatory, e.g.:
+
+.. code-block:: yaml
+
+  # Diagnostics create plots? [true]/false
+  write_plots: true
+  # Diagnositcs write NetCDF files? [true]/false
+  write_netcdf: true
+
+The ``write_plots`` setting is used to inform ESMValTool diagnostics about your
+preference for creating figures. Similarly, the ``write_netcdf`` setting is a
+boolean which turns on or off the writing of netCDF files by the diagnostic
+scripts.
+
+.. code-block:: yaml
+
+  # Auxiliary data directory (used for some additional datasets)
+  auxiliary_data_dir: ~/auxiliary_data
+
+The ``auxiliary_data_dir`` setting is the path to place any required
+additional auxiliary data files. This is necessary because certain
+Python toolkits, such as cartopy, will attempt to download data files at run
+time, typically geographic data files such as coastlines or land surface maps.
+This can fail if the machine does not have access to the wider internet. This
+location allows the user to specify where to find such files if they can not be
+downloaded at runtime.
+
+.. warning::
+
+   This setting is not for model or observational datasets, rather it is for
+   data files used in plotting such as coastline descriptions and so on.
+
+A detailed explanation of the data finding-related sections of the
+``config-user.yml`` (``rootpath`` and ``drs``) is presented in the
+:ref:`data-retrieval` section. This section relates directly to the data
+finding capabilities  of ESMValTool and are very important to be understood by
+the user.
+
+.. note::
+
+   You choose your config.yml file at run time, so you could have several of
```

Comment:
Should be "config-user.yml" here instead of "config.yml"?

Reply:
```suggestion
   You choose your ``config-user.yml`` file at run time, so you could have several of
```